perllib/FixMyStreet/Cobrand/Bexley.pm

```diff
@@ -131,4 +131,31 @@ sub open311_post_send {
     $sender->send($row, $h);
 }
 
+sub dashboard_export_problems_add_columns {
+    my $self = shift;
+    my $c = $self->{c};
+
+    my %groups;
+    if ($c->stash->{body}) {
+        %groups = FixMyStreet::DB->resultset('Contact')->active->search({
+            body_id => $c->stash->{body}->id,
+        })->group_lookup;
```

Comment:
This could return an array ref if it's been set through the admin.

Reply:
...or if multi groups ever switched on, Open311 would do it too, yeah, might as well try and cope, ta.